### ISSUE TITLE
Feature Request: Label Holder (Shelf) and Scoop for EcoBin

### ISSUE THREAD

**Issue by cyberthom42** [Stu142/FreeCAD-Gridfinity-Workbench#23]
Great work! Could you add an option to configure label holder and Scoop for the EcoBin?

**Comment by Stu142** [comment]
Yup makes sense. Labelshelf shouldn't be too bad, but the scoop is harder with the shape of the bottom of the bin.

might be a while before I get to it but I'll get the label shelf in at least at some point
